packages/backend/server/src/modules/features/types.ts

```diff
@@ -1,26 +1,42 @@
-import type { Prisma } from '@prisma/client';
+import { getHostname } from 'tldts';
```

Comment:
use `node:url` here, the builtin ada url parser is extremely fast: https://github.com/ada-url/ada